server/models/proposal.js

```diff
@@ -22,6 +23,7 @@ class Proposal extends AnchorModel {
       reviewComment: null,
       reviewDate: null,
       finalReviewerId: null,
+      hasChild: false,
```

Comment:
Remove this field. We don't really need this field. And you need to refactor some parts of the the code that are using this field. if we want to know whether a proposal has a child or not you can filter like {groupId: proposal._id.toString()}